examples/testplans/release/release.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "Release Plan",
+    "description": "Release Plan to be performed during the release itself",
+
+    "tests": [
+         {
+	 "name": "Version Bump: Version number",
+         "description": "For the Avocado versioning, two files need to receive a manual version update: `VERSION` and `python-avocado.spec`."
+	 },
+         {
+	 "name": "Version Bump: Propagate to plugins",
+         "description": "Run the command: `make propagate-version` to propagate this change to all optional and “linkabe” plugins sharing the parent dir (eg. avocado-vt)."
+	 },
+         {
+	 "name": "Version Bump: Propagate to plugins",
+         "description": "Run the command: `make propagate-version` to propagate this change to all optional and “linkabe” plugins sharing the parent dir (eg. avocado-vt)."
+	 },
+         {
+	 "name": "Version Bump: Commit the bump",
+         "description": "Commit the version bump. Don't forget to commit the changes of 'linked' plugins as they live in different repositories."
+	 },
+         {
+	 "name": "Version Bump: Tag the repository",
+         "description": "When everything is in good shape, commit the version changes and tag that commit in master with:\n\n`git tag -u $(GPG_ID) -s $(RELEASE) -m 'Avocado Release $(RELEASE)'`"
+	 },
+         {
+	 "name": "Version Bump: Push the tag",
+         "description": "The tag should be pushed to the GIT repository with:\n\n`git push --tags`"
+	 },
+         {
+	 "name": "Build RPMs",
```

Comment:
There are other make targets for releases: `make rpm-release`

Reply:
ok, added a new step.

Reply:
I meant using `make rpm-release` *instead* of `make rpm`.